File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/Services/ServiceAccess.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models.Services
{
    public class ServiceAccess
    {
        [XmlAttribute("description")]
        public string Description { get; set; }
        [XmlAttribute("serviceIdentifier")]
        public string ServiceIdentifier { get; set; }
        [XmlAttribute("status")]
        public string Status { get; set; }
        [XmlAttribute("statusDesc")]
        public string StatusDesc { get; set; }
        [XmlAttribute("readOnly")]
        public string ReadOnly { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/ServiceAccess.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "ServiceAccess")
@XmlAccessorType(XmlAccessType.FIELD)
data class ServiceAccess(
        @XmlAttribute
        var description: String = "default description",
        @XmlAttribute
        var serviceIdentifier: String = "default serviceIdentifier",
        @XmlAttribute
        var status: String = "default status",
        @XmlAttribute
        var statusDesc: String = "default statusDesc",
        @XmlAttribute
        var readOnly: String = "default readOnly"
)